allensdk/brain_observatory/behavior/behavior_project_cache/project_apis/data_io/behavior_neuropixels_project_cloud_api.py

```diff
@@ -44,10 +52,16 @@ def get_behavior_session(
         -------
         BehaviorSession
         """
-        row = self._behavior_session_table.query(
-            f"behavior_session_id=={behavior_session_id}"
-        )
-        if row.shape[0] != 1:
+        try:
```

Comment:
the logic to check if the id is in the index exactly 1 time is repeated. Could you write a function to avoid duplicating code?